src/softfloat.h

```diff
@@ -115,3 +115,17 @@ static inline void set_rounding_mode(riscv_t *rv)
     if (frm == 0b100)
         softfloat_roundingMode = softfloat_round_near_maxMag;
 }
+
+static inline void set_static_rounding_mode(uint8_t rm)
+{
+    if (rm == 0b000)
```

Comment:
Rewrite with switch-case. In addition, consider the use of `__UNREACHABLE` when possible.